packages/playwright/src/executors/playwright/playwright.impl.ts

```diff
@@ -111,6 +111,17 @@ function createArgs(
   }
 
   for (const key in rest) {
+    if (key === 'baseUrl') {
+      // baseUrl should not exist as dictated by the Schema
+      // However, it does exist in some existing workspaces
+      console.warn(
+        "'baseUrl' is not a valid option for @nx/playwright:playwright. Please remove this from your project.json."
+      );
+      console.warn(
+        "You can configure 'baseUrl' in the 'playwright.config.ts' file."
+      );
+      continue;
+    }
```

Comment:
I don't think we need to validate this here. It's on users to provide the right extra args for Playwright and those should be validated/rejected by Playwright, not us.
Existing workspaces will get the option removed by the migration.